bridge-api/paths/studies/v3_studies_self_synapseProject.yml

```diff
@@ -12,7 +12,15 @@ post:
     security:
         - BridgeSecurity: []
     parameters:
-        - $ref: ../../index.yml#/parameters/synapseUserId
+        - name: body
+          required: true
+          in: body
+          schema:
+            type: array
+            description: A list of synapse user ids to add as admins into new Project and Team
```

Comment:
A list of synapse user IDs to add as admins of the created project and access team